### DISCUSSION TITLE
3 devices bricked with the transitional ATC firmware. Do not recommend.

### DISCUSSION THREAD

**Discussion by ALERTua** [devbis/z03mmc#81] [General]
Today I bricked 3 LYWSD03MMC devices after flashing them with the transitional ATC firmware. Do not recommend it if you don't have USB-TTL with a 1K resistor and don't know how to solder, like me.

**Comment by morenod** [comment]
Can you confirm pin numbers for tx and Rx? From the picture, I assume one of them is P14, which is the other? P7?

**Reply by devbis** [reply]
They are both to p14, but tx through a resistor. It is used to control the flow while flashing

**Comment by rwronski** [comment]
I've also bricked my device using the transitional firmware ATC_ota_40000.bin. The display is blank, reinserting the battery didn't help either. See log below:

21:53:27: Searching for devices
21:53:59: Get Advertising MAC (Web Experimental Features!)
21:53:59: Connecting to: LYWSD03MMC
21:54:11: MiVer5, DevID: 0x055B-LYWSD03MMC, FnCnt: 1, CtrID: 0x5830 Not bound, Standard certification, MAC: A4C13804DA34, Capability: 0x28 – front binding, IO: 0x0001
21:54:21: Hardware Revision String: B1.4
21:54:21: Software Revision String: 0130
21:54:21: Firmware Revision String: 1.0.0_0130
21:54:21: Detected Mi device
21:54:21: Connected
21:54:41: Activating now, please wait…
21:54:45: Registration successful
21:54:45: Send Login, please wait…
21:54:45: Received device infos are correct
21:54:45: Login successful
21:56:26: File: ATC_ota_40000.bin
21:56:26: File size: 82628 bytes
21:56:26: Count: 5165
21:56:55: Start DFU
21:58:17: Update done after 80.452 seconds
21:58:23: Disconnected.
